docs/best_practice/crate_ami.txt

```diff
@@ -12,7 +12,11 @@ convention is as follows:
   ``Crate-AWS-AMI-<VERSION>-<REVISION>``
 
 The placeholder ``<VERSION>`` is the Crate version in format x.y.z and
-``<REVISION`` describes its build revision.
+``<REVISION>`` describes its build revision. Crate AWS AMI is based on Amazon
+Linux (`Amazon Linux AMI`_) and it's pre-Installed packages are:
```

Comment:
~~it's pre-Installed packages~~
and comes with Java 8 (<exact package name>) and Crate (<exact package name>) pre-installed.